QuizApp/App/Screens/Quiz session/QuizSessionViewController.swift

```diff
@@ -0,0 +1,150 @@
+import UIKit
+
+final class QuizSessionViewController: UIViewController {
+
+    enum Constants {
+
+        static let topMargin = 20
+        static let horizontalMargin = 20
+
+    }
+
+    private var questionNumberLabel: UILabel!
+    private var progressView: ProgressView!
+    private var questionTextLabel: UILabel!
+    private var buttonsStackView: UIStackView!
+    private var buttons: [UIButton]!
+    private var viewModel: QuizSessionViewModel!
+
+    var questionNumber = 1 {
+        didSet {
+            update()
+        }
+    }
+
+    init(viewModel: QuizSessionViewModel) {
+        self.viewModel = viewModel
+        super.init(nibName: nil, bundle: nil)
+    }
+
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        createViews()
+        styleViews()
+        defineLayoutForViews()
+        setupNavigationBar()
+        update()
+    }
+
+}
+
+extension QuizSessionViewController: ConstructViewsProtocol {
+
+    func createViews() {
+        questionNumberLabel = UILabel()
+        view.addSubview(questionNumberLabel)
+
+        progressView = ProgressView(numberOfQuestions: viewModel.quiz.numberOfQuestions)
+        view.addSubview(progressView)
+
+        questionTextLabel = UILabel()
+        view.addSubview(questionTextLabel)
+
+        buttonsStackView = UIStackView()
+        view.addSubview(buttonsStackView)
+
+        buttons = [
+            UIButton(),
+            UIButton(),
+            UIButton(),
+            UIButton()
```

Comment:
Check if there is a fixed number of answers or variable.

Reply:
I've checked, there were mostly 4 answers, but in one case, there were only 2, true or false. I will correct this in the next PR, the one I'm already doing, quiz session API. There I have created models needed for the quiz session, so I will be able to grab the number of answers.